src/ansys/mapdl/core/krylov.py

```diff
@@ -50,7 +51,7 @@ def __init__(self, mapdl):
         if not isinstance(mapdl, MapdlGrpc):  # pragma: no cover
             raise TypeError("``mapdl`` must be a MapdlGrpc instance")
         self._mapdl_weakref = weakref.ref(mapdl)
-        self.mm = self._mapdl.math
+        self.mm = AnsMath(self._mapdl)
```

Comment:
I rather keep `self._mapdl.math` so you "reuse" the already instantiated `AnsMath` instance.